tutorials/parser/src/main/java/com/datatorrent/tutorial/jsonparser/Campaign.java

```diff
@@ -0,0 +1,24 @@
+package com.datatorrent.tutorial.jsonparser;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Campaign
+{
+  public int adId;
+  public String campaignName;
+  @JsonProperty("budget")
+  public double campaignBudget;
+  public boolean weatherTargeting;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy/MM/dd")
+  public Date startDate;
+  
```

Comment:
No getter/setter required for this? What happans when the variables are private? Can you have a proper POJO instead?